pkg/cmd/gtctl/cluster/create/create.go

```diff
@@ -57,8 +57,11 @@ type createClusterCliOptions struct {
 	AlwaysDownload     bool
 
 	// Common options.
-	Timeout int
-	DryRun  bool
+	Timeout     int
+	DryRun      bool
+	Set         string
```

Comment:
suggestion: maybe we just need one `Set`(**to make cli more clean**) and use the some format to differentiate different components? For example:

```
gtctl cluster create mycluster \
--set cluster.frontend.replicas=1 \
--set etcd.storageClassName=standard \
--set operator.replicas=2
```

For the above example, I use `cluster`/`etcd`/`operator` as the identifier of the first part of the string.

Reply:
great idea, also should be pointed out in its usage